pkg/fscrypt/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.12.4-alpine
+
+RUN apk add --no-cache git=2.20.1-r0 gcc=8.3.0-r0 linux-headers=4.18.13-r1 \
+                       libc-dev=0.7.1-r0 make=4.2.1-r2 linux-pam-dev=1.3.0-r0 \
+                       m4=1.4.18-r1
+
+RUN mkdir -p /go/src/github.com/google
+WORKDIR /go/src/github.com/google
+RUN git clone https://github.com/google/fscrypt
+WORKDIR /go/src/github.com/google/fscrypt
+RUN make && make install
```

Comment:
This will be produce quite a big container since all of the go binaries and source go will still be there. You need to use two stage Docker build to only copy what you need into a resulting container. Please take a look at how rkt and rkt-stage1 packages do this. @bharani-zededa can help if you still have questions.

Reply:
I've made it into a 2-stage build, and updated the PR.